db/migrate/20200408173214_add_foreign_key_constraints.rb

```diff
@@ -0,0 +1,19 @@
+class AddForeignKeyConstraints < ActiveRecord::Migration[6.0]
+  def change
+    add_foreign_key :activities, :organisations, on_delete: :restrict
```

Comment:
Ensure that organisations cannot be deleted whilst they are still referenced by an activity.